ecosystem/typescript/sdk_v2/src/crypto/ed25519.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { Deserializer, Serializer } from "../bcs";
+import { Hex } from "../core";
+import { HexInput } from "../types";
+
+export class Ed25519PublicKey {
+  static readonly LENGTH: number = 32;
+
+  readonly value: Hex;
```

Comment:
`data` for consistency. 
```suggestion
  private readonly data: Hex;
```